modules/sql-engine/src/integrationTest/sql/etc/test_not_null_contraint.test

```diff
@@ -0,0 +1,158 @@
+# name: test/sql/etc/test_not_null_constraint.test
+# description: E141-01 (Basic integrity constraints, NOT NULL constraints)
+# group: [etc]
```

Comment:
maybe better to create a group (together with a folder) with a name like `constraints`?